components/notebook-controller/Makefile

```diff
@@ -1,6 +1,6 @@
 
 # Image URL to use all building/pushing image targets
-IMG ?= gcr.io/kubeflow-images-public/notebook-controller
+IMG ?= quay.io/4n4nd/notebook-controller
```

Comment:
Let's choose a common quay.io org.  I know of [opendatahub](https://quay.io/organization/opendatahub).  Does AICoE have one?

Reply:
also, can't we just override this with an env var?

Reply:
> also, can't we just override this with an env var?
yeah I didn't really think about it